### PR TITLE
Sort qubit operation tests

### PR DESCRIPTION
**Pull request by albi3ro**
The current qubit operation testing file is over 3,000 lines long.  This makes it difficult to figure out where to put new tests and find existing ones.

Recently, the qubit operations file was divided into several categories.  This performs the same division on the tests.

### CODE REVIEW

File: tests/ops/qubit/test_qubit_arithmetic_ops.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_arithmetic_ops.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_state_prep.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_state_prep.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_qchem_ops.py [hunk 3]
```diff
@@ -0,0 +1,562 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_qchem_ops.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_parametric_ops.py [hunk 4]
```diff
@@ -0,0 +1,1364 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_parametric_ops.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_ops.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_ops.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_observablers.py [hunk 6]
```diff
@@ -0,0 +1,376 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_observablers.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_non_parametric_ops.py [hunk 7]
```diff
@@ -0,0 +1,710 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_non_parametric_ops.py:1]:
```suggestion
# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

File: tests/ops/qubit/test_qubit_observablers.py [hunk 8]
```diff
@@ -0,0 +1,376 @@
...
+
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_observablers.py:2]: Can't suggest, but the name of the file needs adjustment: `test_qubit_observablers.py` -> `test_qubit_observables.py`.

**Review by antalszava — CHANGES_REQUESTED** [review]
@albi3ro overall this is looking great! :slightly_smiling_face: A couple of small things to adjust. One change that would be great: the `qubit` in the filename seems to be redundant (as files are already in a `qubit` folder). Could we have the test files be renamed?

File: tests/ops/qubit/test_qubit_ops.py [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+Unit tests for the available built-in discrete-variable quantum operations. Only tests over multiple
+types of operations should exist in this file. Type-specific tests should go in the more specific file.
+"""
+import pytest
+import pennylane as qml
+
+from gate_data import I
+
+
+class TestOperations:
```
Comment (antalszava) [inline tests/ops/qubit/test_qubit_ops.py:24]: @albi3ro wondering about this file, would the content here belong into the parametric test file?

Comment (albi3ro) [reply tests/ops/qubit/test_qubit_ops.py:24]: Because it's also testing non-parametrized gates as well.  We could just split the test in two...

Comment (antalszava) [reply tests/ops/qubit/test_qubit_ops.py:24]: Yep, I think that could work. It feels a bit strange to have this file even after the split of files.

**Review by antalszava — APPROVED** [review]
Looks good to me! :100: So nice to see this being tidied up, thank you. :slightly_smiling_face:

**Pull request merged**